scripts/core/ui/components/video.tsx

```diff
@@ -32,6 +32,7 @@ export class VideoComponent extends React.PureComponent<IProps> {
             if (rend.mimetype == null) {
                 continue;
             } else if (rend.mimetype.toLowerCase() === 'application/x-mpegurl') {
+                // HLSVideoComponent can handle only one stream url
```

Comment:
Sorry to bother you, but could you clarify the comment to state that "Only one instance of `HLSVideoComponent` can be rendered in the page at any given moment.`?

When you refer to the component being able to only handle one stream URL it wouldn't be incorrect to assume that you can render 5 instances and handle 5 streams this way.

Reply:
You can render 5 HLSVideoComponent and theoretically, they will work simultaneously. I've tried with 2. 
When I said `it can handle only one stream URL at a period of time` I did not know that it can actually do this :) But you can provide only one source url to the component which is documented. 
P.S. You are not bothering me :) It's good to have such moments to be clear for other devs.

Reply:
If multiple `HLSVideoComponent`s can work simultaneously, will you then remove `videoRenditions` variable and implement the `if` for deciding which video component to use at line 57 so multiple instances of `HLSVideoComponent` could be used at once?

Reply:
I'm not sure if we actually need this. In this case, it will render more than one `HLSVideoComponents`. 
If video is not a stream we render only one player even if we have multiple renditions. 
The difference is that for default video (not stream) we don't use `<video src=".."/>` we use 
```
<video>
<source ..>
<source ..>
</vidoe>
```
which at the end renders only one instance of player.

Reply:
Can you add a comment that `VideoComponent` is only used when `item.type === 'video'`?

I thought it could be used to display multiple different videos in an article, that's why I was suggesting the mapping approach.

Reply:
done